src/System.Management.Automation/help/HelpUtils.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Help;
+using Microsoft.PowerShell.Commands;
+
+namespace System.Management.Automation
+{
+    internal class HelpUtils
+    {
+        private static string userScopeRootPath = null;
+        private static string userHomeHelpPath = null;
+
+
+        /// <summary>
+        /// Get the path to $HOME
+        /// </summary>
+        internal static string GetUserHomeHelpSearchPath()
+        {
+            if (userScopeRootPath == null)
```

Comment:
I don't see `userScopeRootPath` gets used except for calculating `userHomeHelpPath`. Maybe it shouldn't be made a static variable.

Reply:
Fixed.